tests/unit/regression/lars_unittest.cc

```diff
@@ -228,4 +234,98 @@ TEST(LeastAngleRegression, lars_N_LessThan_D)
 	SG_UNREF(features);
 	SG_UNREF(labels);
 }
+
+TEST(LeastAngleRegression, cholesky_insert)
+{
+	
+	SGMatrix<float64_t> mat=SGMatrix<float64_t>(6,4);
+	SGMatrix<float64_t> matn=SGMatrix<float64_t>(6,5);
+
+	SGVector<float64_t> vec=SGVector<float64_t>(6);
+	vec.random(0.0,1.0);
+	Map<VectorXd> map_vec(vec.vector, vec.size());	
+
+	for (index_t i=0; i<6; i++)
+	{	
+		for (index_t j=0; j<4; j++)
+		{
+			mat(i,j)=CMath::random(0.0,1.0);
+			matn(i,j)=mat(i,j);
+		}
+	}
+	for (index_t i=0 ;i<6; i++)
```

Comment:
please use variables instead of magical constants like 6 and 4...
so lie define:
```
int32_t mat_rows = 6;
int32_t mat_cols = 6;
...
```
after that you can refer to those variables + changing the unit test from 6 to 60 would be easy :P